### PR TITLE
Frontend improvements

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

- [x] Feature: Duplicate study. Frontend ready
- [x] Inputs/outputs in Settings View
  - Inputs/Ouputs
    - Label + Description go as info tooltip
    - Value goes as label in tree. "-" if not available
    - Chevrons on side panels to show they are collapsible
  - Settings
    - Description goes next to label as info tooltip
- [x] Bug: Stop spinning
- [x] Bug: Dashed connections
- [x] Bug: Data Browser was quite buggy
- [x] Refactoring: infoWHint decoupled from fieldWHint class
- [x] UX: improve File Picker's experience
  - Auto select file when adding it within the File Picker
  - Refactoring Study Model to start supporting Output objects. FP uses it.

Inputs/outputs in Settings View:
image

Clone study. Frontend ready:
clone

## Related issue/s
frontend for https://github.com/ITISFoundation/osparc-issues/issues/402
closes https://github.com/ITISFoundation/osparc-issues/issues/260
closes https://github.com/ITISFoundation/osparc-simcore/issues/1742

### CODE REVIEW

File: services/web/client/source/class/osparc/desktop/SidePanel.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ qx.Class.define("osparc.desktop.SidePanel", {
     collapsed: {
       init: false,
       check: "Boolean",
+      event: "changeCollapsed",
```
Comment (KZzizzle) [inline services/web/client/source/class/osparc/desktop/SidePanel.js:52]: thank you!!

File: services/web/client/source/class/osparc/desktop/WorkbenchView.js [hunk 2]
```diff
@@ -303,7 +303,7 @@ qx.Class.define("osparc.desktop.WorkbenchView", {
         const nodeId = e.getData();
         this.__removeNode(nodeId);
       }, this);
-      this.__sidePanel.addOrReplaceAt(new osparc.desktop.PanelView(this.tr("Service tree"), nodesTree), 0, {
+      this.__sidePanel.addOrReplaceAt(new osparc.desktop.PanelView(this.tr("Nodes tree"), nodesTree), 0, {
```
Comment (KZzizzle) [inline services/web/client/source/class/osparc/desktop/WorkbenchView.js:306]: I guess that's what we're calling them now =)

Comment (odeimaiz) [reply services/web/client/source/class/osparc/desktop/WorkbenchView.js:306]: And we call "Sharing" to "Permissions"

File: services/web/client/source/class/osparc/theme/mixin/Color.js [hunk 3]
```diff
@@ -2,7 +2,8 @@ qx.Theme.define("osparc.theme.mixin.Color", {
   colors: {
     "activitytree-background-cpu": "#2C7CCE",
     "activitytree-background-memory": "#358475",
-    "ready-green": "#33925A",
+    // "ready-green": "#33925A",
+    "ready-green": "#58A6FF", // It is not really green because of reasons
```
Comment (KZzizzle) [inline services/web/client/source/class/osparc/theme/mixin/Color.js:6]: =D amazing!

**Comment by KZzizzle** [comment]
Request, when a background process is started, could the "x" icon be replaced with something like: 
image
This is because the x is usually used to close windows (e.g. close the popup) as opposed to ending the process. I would have suggested the same icon as the stop button but a square is maybe a bit ambiguous.

**Review by KZzizzle — APPROVED** [review]

File: services/web/client/source/class/osparc/dashboard/StudyBrowser.js [hunk 4]
```diff
@@ -628,6 +639,40 @@ qx.Class.define("osparc.dashboard.StudyBrowser", {
...
+      cloningStudyCard.subscribeToFilterGroup("sideSearchFilter");
+      this.__userStudyContainer.addAt(cloningStudyCard, 1);
+
+      const params = {
+        url: {
+          projectId: studyData["uuid"]
+        },
+        data: osparc.utils.Utils.getClientSessionID()
+      };
+      osparc.data.Resources.fetch("studies", "duplicate", params)
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/dashboard/StudyBrowser.js:662]: why are you calling it ``duplicate`` in the API and ``clone`` in the UI? At this point we are still in time to call things in one way :-) ... or we will end up again with the study/project/node **unnecessary** dilemmas. Please select the one you like the most but only one.

**Review by pcrespov — CHANGES_REQUESTED** [review]
better and better but ... :-)

File: services/web/client/source/class/osparc/dashboard/StudyBrowser.js (hunk 4 above)
Comment (odeimaiz) [reply services/web/client/source/class/osparc/dashboard/StudyBrowser.js:662]: ```duplicate``` it is, then!

**Review by mguidon — APPROVED** [review]
nice. Maybe you should start grouping the menu items in the study?

**Review by pcrespov — APPROVED** [review]
thx!

**Pull request merged**